host/bcm283x/gpio.go

```diff
@@ -266,6 +266,16 @@ func (p *Pin) Out(l gpio.Level) error {
 		return err
 	}
 	// Change output before changing mode to not create any glitch.
+	p.FastOut(l)
+	p.setFunction(out)
+	return nil
+}
+
+// FastOut sets a pin output level with Absolutely No error checking.
+//
+// Out() Must be called once first, then FastOut() can be used for minimal CPU
```

Comment:
what happens of Out isn't called first?

Reply:
Undefined behavior. Updated the comment to be clearer, wdyt?